### DISCUSSION TITLE
Adding wifi and blueman (bluetooth) icons on Picom

### DISCUSSION THREAD

**Discussion by ankeorum** [yshui/picom#1319] [Configuraiton]
Hi everybody!

First of all, thank you for this feature, it is so nice! Secondly, I am trying to configure and create two more options (they could be next to the volume icon or a new option) which are blueman (the bluetooth device manager tool) and the wifi management tool too. This will be so useful for when I am travelling and need connecting new devices or the pentesting device to new networks. 

Thank you so much!

**Comment by absolutelynothelix** [comment]
i believe you're confusing picom with something else. picom is a compositor that makes your windows look a little bit better by e.g. drawing shadows.

**Reply by ankeorum** [reply]
I may have confused it with polybar, you are right; apologises
